lib/blocklist.js

```diff
@@ -0,0 +1,48 @@
+/* Any copyright is dedicated to the Public Domain.
```

Comment:
I'm not sure, some files have Public Domain, some have MPL, @rfk do we have a stance on which?